my_class.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './module_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to implement an `each` method of your own instead of using each method of the ruby language, this will help you understand how the method is created by implementing it from scratch